production/ksonnet/loki/gateway.libsonnet

```diff
@@ -43,22 +43,33 @@
             listen               80;
             auth_basic           “Prometheus”;
             auth_basic_user_file /etc/nginx/secrets/.htpasswd;
-            proxy_set_header     X-Scope-OrgID 1;
```

Comment:
Do we need to remove this one ? Sorry I'm a bit lost.

Reply:
Sorry I misunderstood.
I fixed https://github.com/grafana/loki/pull/1689/commits/7c535a273e9ef7525af6dbb94b0068e18064a25f